---
title: 'Homework "0"'
output: rmarkdown::html_vignette
vignette: >
  %\VignetteIndexEntry{Homework "0"}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>"
)
```
First things first.  Anything you see in a gray box represents something
you can copy and paste in the R console (bottom left panel in RStudio,
or the whole panel in R). Anything you see that starts with a "#"
sign is just a comment.  You can copy and paste those lines to R
(or RStudio), but they will not do anything.

Now for a definite first things first.  Whenever you do homework
assignments you will **have** to copy and paste the line below so that
you have the necessary package working.  It is called Anth241Hmwk.

```{r eval=F}
# Always start your homework by copying and pasting the line below
library(Anth241Hmwk)
```

The code below is all you need for Homework "0".  It is called Homework
"0" because it is really just a test to make sure that you have
installed R/RStudio and that you know your way around.
There are two "problems" that you can run.  You can do either
one, or both if you want.  For the first one, you will produce
two plots, both of which you should copy to the \*.doc or \*.docx
file that you will submit.


To put the horse way before the cart, what the code below is doing is
finding the "heterozygosity" for a locus where there are \emph{n}
different alleles (variants of genes) where the alleles are equally
frequent (so, all have a frequency of $\frac {1}{n}$).  "Heterozygosity"
is the frequency of heterozygosity (individuals with different allelic
variants, like AB genotype in the ABO blood system).  Under
Hardy-Weinberg (something we will learn about), the heterozygosity
is $H_n=1-\frac{1}{n}$. The code is finding $H_n$ across various values
of \emph{n} and plotting these against the number of alleles, \emph{n}.

```{r, eval=FALSE}
# Copy the line below that does not have a # sign, paste to console 
# and "enter" or "return", paste and copy plot to your *.doc or *.docx
# file.  This script is plotting heterozygosity from 5 to 50 alleles.
# Note: you can actually copy and paste lines that start with a # sign
# as such lines are just comments that will not do anything.
Fig3_1()
```

```{r, eval=FALSE}
# Do the same for the line below
Fig3_1(1,3)
```

Here's an alternative problem, and actually one that you may find
more interesting (if you are into GofT).  That's "Game of Thrones."
It draws a pedigree and calculates the inbreeding coefficients.
Inbreeding coefficients are something we will learn about later
in the course, but as you might guess, an individual with an
inbreeding coefficient equal to zero is not inbred.

```{r eval=FALSE}
# Run this and copy and paste the inbreeding coefficients listed
# in the R console to your document and copy and paste to you document
# the pedigree that is in the graphic window
make_ped2()
```
