<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />

<meta name="viewport" content="width=device-width, initial-scale=1" />



<title>Homework 1</title>

<script>// Pandoc 2.9 adds attributes on both header and div. We remove the former (to
// be compatible with the behavior of Pandoc < 2.8).
document.addEventListener('DOMContentLoaded', function(e) {
  var hs = document.querySelectorAll("div.section[class*='level'] > :first-child");
  var i, h, a;
  for (i = 0; i < hs.length; i++) {
    h = hs[i];
    if (!/^h[1-6]$/i.test(h.tagName)) continue;  // it should be a header h1-h6
    a = h.attributes;
    while (a.length > 0) h.removeAttribute(a[0].name);
  }
});
</script>

<style type="text/css">
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
</style>



<style type="text/css">
code {
white-space: pre;
}
.sourceCode {
overflow: visible;
}
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
{ counter-reset: source-line 0; }
pre.numberSource code > span
{ position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
{ content: counter(source-line);
position: relative; left: -1em; text-align: right; vertical-align: baseline;
border: none; display: inline-block;
-webkit-touch-callout: none; -webkit-user-select: none;
-khtml-user-select: none; -moz-user-select: none;
-ms-user-select: none; user-select: none;
padding: 0 4px; width: 4em;
color: #aaaaaa;
}
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa; padding-left: 4px; }
div.sourceCode
{ }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } 
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } 
code span.at { color: #7d9029; } 
code span.bn { color: #40a070; } 
code span.bu { color: #008000; } 
code span.cf { color: #007020; font-weight: bold; } 
code span.ch { color: #4070a0; } 
code span.cn { color: #880000; } 
code span.co { color: #60a0b0; font-style: italic; } 
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } 
code span.do { color: #ba2121; font-style: italic; } 
code span.dt { color: #902000; } 
code span.dv { color: #40a070; } 
code span.er { color: #ff0000; font-weight: bold; } 
code span.ex { } 
code span.fl { color: #40a070; } 
code span.fu { color: #06287e; } 
code span.im { color: #008000; font-weight: bold; } 
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } 
code span.kw { color: #007020; font-weight: bold; } 
code span.op { color: #666666; } 
code span.ot { color: #007020; } 
code span.pp { color: #bc7a00; } 
code span.sc { color: #4070a0; } 
code span.ss { color: #bb6688; } 
code span.st { color: #4070a0; } 
code span.va { color: #19177c; } 
code span.vs { color: #4070a0; } 
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } 
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>




<style type="text/css">body {
background-color: #fff;
margin: 1em auto;
max-width: 700px;
overflow: visible;
padding-left: 2em;
padding-right: 2em;
font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
font-size: 14px;
line-height: 1.35;
}
#TOC {
clear: both;
margin: 0 0 10px 10px;
padding: 4px;
width: 400px;
border: 1px solid #CCCCCC;
border-radius: 5px;
background-color: #f6f6f6;
font-size: 13px;
line-height: 1.3;
}
#TOC .toctitle {
font-weight: bold;
font-size: 15px;
margin-left: 5px;
}
#TOC ul {
padding-left: 40px;
margin-left: -1.5em;
margin-top: 5px;
margin-bottom: 5px;
}
#TOC ul ul {
margin-left: -2em;
}
#TOC li {
line-height: 16px;
}
table {
margin: 1em auto;
border-width: 1px;
border-color: #DDDDDD;
border-style: outset;
border-collapse: collapse;
}
table th {
border-width: 2px;
padding: 5px;
border-style: inset;
}
table td {
border-width: 1px;
border-style: inset;
line-height: 18px;
padding: 5px 5px;
}
table, table th, table td {
border-left-style: none;
border-right-style: none;
}
table thead, table tr.even {
background-color: #f7f7f7;
}
p {
margin: 0.5em 0;
}
blockquote {
background-color: #f6f6f6;
padding: 0.25em 0.75em;
}
hr {
border-style: solid;
border: none;
border-top: 1px solid #777;
margin: 28px 0;
}
dl {
margin-left: 0;
}
dl dd {
margin-bottom: 13px;
margin-left: 13px;
}
dl dt {
font-weight: bold;
}
ul {
margin-top: 0;
}
ul li {
list-style: circle outside;
}
ul ul {
margin-bottom: 0;
}
pre, code {
background-color: #f7f7f7;
border-radius: 3px;
color: #333;
white-space: pre-wrap; 
}
pre {
border-radius: 3px;
margin: 5px 0px 10px 0px;
padding: 10px;
}
pre:not([class]) {
background-color: #f7f7f7;
}
code {
font-family: Consolas, Monaco, 'Courier New', monospace;
font-size: 85%;
}
p > code, li > code {
padding: 2px 0px;
}
div.figure {
text-align: center;
}
img {
background-color: #FFFFFF;
padding: 2px;
border: 1px solid #DDDDDD;
border-radius: 3px;
border: 1px solid #CCCCCC;
margin: 0 5px;
}
h1 {
margin-top: 0;
font-size: 35px;
line-height: 40px;
}
h2 {
border-bottom: 4px solid #f7f7f7;
padding-top: 10px;
padding-bottom: 2px;
font-size: 145%;
}
h3 {
border-bottom: 2px solid #f7f7f7;
padding-top: 10px;
font-size: 120%;
}
h4 {
border-bottom: 1px solid #f7f7f7;
margin-left: 8px;
font-size: 105%;
}
h5, h6 {
border-bottom: 1px solid #ccc;
font-size: 105%;
}
a {
color: #0033dd;
text-decoration: none;
}
a:hover {
color: #6666ff; }
a:visited {
color: #800080; }
a:visited:hover {
color: #BB00BB; }
a[href^="http:"] {
text-decoration: underline; }
a[href^="https:"] {
text-decoration: underline; }

code > span.kw { color: #555; font-weight: bold; } 
code > span.dt { color: #902000; } 
code > span.dv { color: #40a070; } 
code > span.bn { color: #d14; } 
code > span.fl { color: #d14; } 
code > span.ch { color: #d14; } 
code > span.st { color: #d14; } 
code > span.co { color: #888888; font-style: italic; } 
code > span.ot { color: #007020; } 
code > span.al { color: #ff0000; font-weight: bold; } 
code > span.fu { color: #900; font-weight: bold; } 
code > span.er { color: #a61717; background-color: #e3d2d2; } 
</style>




</head>

<body>




<h1 class="title toc-ignore">Homework 1</h1>



<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" tabindex="-1"></a><span class="co"># Always start your homework by copying and pasting the line below</span></span>
<span id="cb1-2"><a href="#cb1-2" tabindex="-1"></a><span class="fu">library</span>(Anth241Hmwk)</span></code></pre></div>
<div id="hardy-weinberg-equilibrium-for-abo-blood-group" class="section level2">
<h2>1. Hardy-Weinberg equilibrium for ABO blood group</h2>
<p>The script below asks for the number of individuals with type A, B,
AB, and O blood. Only for AB and O phenotypes are the genotypes known
(AB and OO,respectively). Type A includes AA and AO while type B
includes BB and BO. The script uses something called the expectation
maximization algorithm to find the allele frequencies (p [frequency of A
allele], q [frequency of B allele], and r [frequency of O allele]). You
will be asked for phenotype counts and there are suggested phenotype
counts. Use the suggested counts.</p>
<p>The output will be “Obs” and “Exp” which are the observed phenotype
counts and the expected counts if the locus really was in Hardy-Weinberg
equilibrium. You will also get $N (the total number of individuals), $p
(the estimated A allele frequency), $q (the estimated B allele
frequency), and $r (the estimated O allele frequency).</p>
<div id="part-a" class="section level3">
<h3>Part A</h3>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" tabindex="-1"></a><span class="co"># run entering the suggested phenotype counts</span></span>
<span id="cb2-2"><a href="#cb2-2" tabindex="-1"></a><span class="co"># and copy output (Obs &amp; Exp and $N, $p, $q, and $r) and paste</span></span>
<span id="cb2-3"><a href="#cb2-3" tabindex="-1"></a><span class="co"># to your document</span></span>
<span id="cb2-4"><a href="#cb2-4" tabindex="-1"></a><span class="fu">P_56</span>()</span></code></pre></div>
</div>
<div id="part-b" class="section level3">
<h3>Part B</h3>
<p>This time you are going to enter your own counts</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" tabindex="-1"></a><span class="co"># run entering whatever phenotype counts you want</span></span>
<span id="cb3-2"><a href="#cb3-2" tabindex="-1"></a><span class="co"># and copy output (Obs &amp; Exp) and paste</span></span>
<span id="cb3-3"><a href="#cb3-3" tabindex="-1"></a><span class="co"># to your document</span></span>
<span id="cb3-4"><a href="#cb3-4" tabindex="-1"></a>sto<span class="ot">=</span><span class="fu">P_56</span>()</span></code></pre></div>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" tabindex="-1"></a><span class="co"># This time you did not see the $N, $p, $q, and $r. Why is</span></span>
<span id="cb4-2"><a href="#cb4-2" tabindex="-1"></a><span class="co"># that? It is because they got stored in &quot;sto.&quot; To see them</span></span>
<span id="cb4-3"><a href="#cb4-3" tabindex="-1"></a><span class="co"># copy and paste the line below to your console:</span></span>
<span id="cb4-4"><a href="#cb4-4" tabindex="-1"></a>sto</span></code></pre></div>
</div>
<div id="part-c" class="section level3">
<h3>Part C</h3>
<p>Now for some real fun! The reason that we did that storage to “sto”
(my favorite name for storage) is so that we can use the stuff we stored
there to find the expected count of AB blood types, which is <span class="math inline">\(N \times 2 \times p \times q\)</span>. But <a href="https://en.wikipedia.org/wiki/Houston,_we_have_a_problem">“Houston,
we have a problem”</a> There is no <span class="math inline">\(\times\)</span> symbol on the keyboard, nor can we
type “p” or “q” thinking that the keyboard will recognize those as
allele frequencies. Any programming language or scripting language that
you look at will use * (the asterisk symbol, which is on the keyboard)
in place of the <span class="math inline">\(\times\)</span> symbol. So,
the code below will gives us what we want.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" tabindex="-1"></a><span class="co"># Paste this output to your document.  It should be</span></span>
<span id="cb5-2"><a href="#cb5-2" tabindex="-1"></a><span class="co"># the same as from the &quot;expected&quot; value for the AB phenotype</span></span>
<span id="cb5-3"><a href="#cb5-3" tabindex="-1"></a>sto<span class="sc">$</span>N<span class="sc">*</span><span class="dv">2</span><span class="sc">*</span>sto<span class="sc">$</span>p<span class="sc">*</span>sto<span class="sc">$</span>q</span></code></pre></div>
</div>
</div>
<div id="hardy-weinberg-and-probability-values" class="section level2">
<h2>2. Hardy-Weinberg and probability values</h2>
<p>In the next problem we are going to look at genotype counts for a
bi-allelic locus (so AA, AB, and BB genotypes) to see if the genotypes
are in Hardy-Weinberg equilibrium. This script will take the observed
genotype counts and return the expected genotype counts if the genotypes
were in H-W equilibrium. But wait, there’s more. The script will also
return the probability value (“p-value”) that the genotypes are in
Hardy-Weinberg equilibrium. This is found by finding the probability of
getting the observed AA genotype count under H-W equilibrium or any more
extreme count (but will under H-W equilibrium). Aside from just getting
the information in your console, you will also get a graphical
representation of how the probability value is found.</p>
<div id="part-a-1" class="section level3">
<h3>Part A</h3>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" tabindex="-1"></a><span class="co"># Run this and copy/paste console output and graph to your document</span></span>
<span id="cb6-2"><a href="#cb6-2" tabindex="-1"></a><span class="fu">Tab3_2</span>()</span></code></pre></div>
</div>
<div id="part-b-1" class="section level3">
<h3>Part B</h3>
<p>OK, that was running the default values, but now we are going to run
an example where we start in H-W equilibrium (so p-value=1) and move
further and further away from H-W equilibrium by reducing N from 100 to
lower values for the BB genotype count. For each run copy the console
output to your document but <strong>do not</strong> copy and paste the
graphs (that would be a lot of graphs!).</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" tabindex="-1"></a><span class="co"># Copy/paste the two lines below to setup the allele frequencies</span></span>
<span id="cb7-2"><a href="#cb7-2" tabindex="-1"></a>p<span class="ot">=</span><span class="dv">1</span><span class="sc">/</span><span class="dv">3</span></span>
<span id="cb7-3"><a href="#cb7-3" tabindex="-1"></a>q<span class="ot">=</span><span class="dv">1</span><span class="sc">-</span>p <span class="co"># remember, in bi-allelic system q=1-p</span></span></code></pre></div>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" tabindex="-1"></a><span class="co"># Run the line below and copy/paste output to document</span></span>
<span id="cb8-2"><a href="#cb8-2" tabindex="-1"></a><span class="fu">Tab3_2</span>(<span class="dv">100</span><span class="sc">*</span>p<span class="sc">^</span><span class="dv">2</span>,<span class="dv">100</span><span class="sc">*</span><span class="dv">2</span><span class="sc">*</span>p<span class="sc">*</span>q,<span class="dv">100</span><span class="sc">*</span>q<span class="sc">^</span><span class="dv">2</span>) <span class="co"># This is in H-W equil, so p=1</span></span></code></pre></div>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" tabindex="-1"></a><span class="co"># Run the line below and copy/paste output to document</span></span>
<span id="cb9-2"><a href="#cb9-2" tabindex="-1"></a><span class="fu">Tab3_2</span>(<span class="dv">100</span><span class="sc">*</span>p<span class="sc">^</span><span class="dv">2</span>,<span class="dv">100</span><span class="sc">*</span><span class="dv">2</span><span class="sc">*</span>p<span class="sc">*</span>q,<span class="dv">50</span><span class="sc">*</span>q<span class="sc">^</span><span class="dv">2</span>)</span></code></pre></div>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" tabindex="-1"></a><span class="co"># Run the line below and copy/paste output to document</span></span>
<span id="cb10-2"><a href="#cb10-2" tabindex="-1"></a><span class="fu">Tab3_2</span>(<span class="dv">100</span><span class="sc">*</span>p<span class="sc">^</span><span class="dv">2</span>,<span class="dv">100</span><span class="sc">*</span><span class="dv">2</span><span class="sc">*</span>p<span class="sc">*</span>q,<span class="dv">10</span><span class="sc">*</span>q<span class="sc">^</span><span class="dv">2</span>)</span></code></pre></div>
</div>
<div id="part-c-1" class="section level3">
<h3>Part C</h3>
<p>Which of the three runs above gave you the lowest probability value?
(i.e., which of the runs was furthest from being in H-W equilibrium?)
Note that this is due to dropping the number of total individuals for
the BB genotype.</p>
</div>
</div>
<div id="balancingstabilizing-selection" class="section level2">
<h2>3. Balancing/Stabilizing selection</h2>
<p>We will start with the default setting which is for “balancing”
selection. In other words, the heterozygote has the highest relative
fitness (=1.0), while the homozygotes have fitness values that are
greater than 0.0 and less than 1.0. In this case w=c(0.7635,1.0,0.0001)
where the values are the relative fitnesses. In “R-ese” the <span class="math inline">\(c(\cdot)\)</span> is the “combine” function where
the list with commas gets combined into a vector, so the relative
fitness for the AA genotype is 0.7635, for the AS genotype (sickle-cell
trait) is 1.0, and for the SS genotype (sickle-cell disease) is
0.0001.</p>
<div id="part-a-2" class="section level3">
<h3>Part A</h3>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" tabindex="-1"></a><span class="co"># Run and copy/paste the graph to your document</span></span>
<span id="cb11-2"><a href="#cb11-2" tabindex="-1"></a><span class="fu">Fig3_9</span>(<span class="at">p.start=</span><span class="fu">runif</span>(<span class="dv">100</span>))</span></code></pre></div>
</div>
<div id="part-b-2" class="section level3">
<h3>Part B</h3>
<p>What does each line on the graph represent?</p>
</div>
</div>
<div id="heterozygote-disadvantage" class="section level2">
<h2>4. Heterozygote disadvantage</h2>
<p>Now we will look at homozygote advantage, where the relative fitness
of the AA and BB genotypes (the homozygotes) are 1.0 and the
heterozygotes have a relative fitness greater than 0.0 but less than
1.0.</p>
<div id="part-a-3" class="section level3">
<h3>Part A</h3>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" tabindex="-1"></a><span class="co"># Run me and copy/paste the plot to your document</span></span>
<span id="cb12-2"><a href="#cb12-2" tabindex="-1"></a><span class="fu">Fig3_9</span>(<span class="fu">c</span>(<span class="dv">1</span>,.<span class="dv">6</span>,<span class="dv">1</span>),<span class="fu">runif</span>(<span class="dv">100</span>))</span></code></pre></div>
</div>
<div id="part-b-3" class="section level3">
<h3>Part B</h3>
<p>Look up <a href="https://en.wikipedia.org/wiki/Fixed_allele">“fixed
allele”</a> at Wikipedia and explain how this relates to what you see in
the graph (with the realization that either allele, “A” or “B”, can go
to fixation).</p>
</div>
</div>
<div id="adaptive-topography" class="section level2">
<h2>5. Adaptive topography</h2>
<div id="part-a-4" class="section level3">
<h3>Part A</h3>
<p>In this section you will do plots of average relative fitness for the
sickle-cell example and heterozygote disadvantage and paste these graphs
to your homework.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" tabindex="-1"></a><span class="co"># Balanced polymorphism</span></span>
<span id="cb13-2"><a href="#cb13-2" tabindex="-1"></a><span class="fu">Fig3_13</span>()</span></code></pre></div>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" tabindex="-1"></a><span class="co"># Heterozygote disadvantage</span></span>
<span id="cb14-2"><a href="#cb14-2" tabindex="-1"></a><span class="fu">Fig3_13</span>(<span class="fu">c</span>(<span class="dv">1</span>,.<span class="dv">6</span>,<span class="dv">1</span>),<span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">1</span>),<span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">1</span>),<span class="at">tan=</span>.<span class="dv">05</span>)</span></code></pre></div>
</div>
<div id="part-b-4" class="section level3">
<h3>Part B</h3>
<p>In the second plot, if you drew arrows to indicate the direction of
evolution, would they point toward or away from the middle of the plot?
Remember that evolution by selection is about “hill-climbing” in the
adaptive topography. To wit, selection cannot take you “down hill”.</p>
</div>
</div>
<div id="genetic-drift" class="section level2">
<h2>6. Genetic drift</h2>
<p><strong>Note: there is nothing to turn in from this
section</strong></p>
<p>Genetic drift is one of the harder concepts to grasp, so it will
behoove you to look at a number of graphs. The scripts Fig3_17 and
Fig3_20 were written in part so that people could “play around” and look
at a bunch of graphs. You will see graphs like those out of Fig3_17 on
your exam and will be asked to sort them by increasing sub-population
size. Similarly, you will see graphs like Fig3_20 but with generations
out of order and you will be asked to sort them by generation.</p>
<div id="stochastic-simulation-of-genetic-drift" class="section level4">
<h4>Stochastic simulation of genetic drift</h4>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" tabindex="-1"></a><span class="fu">Fig3_17</span>(<span class="dv">10</span><span class="sc">^</span><span class="dv">0</span>,<span class="at">jit=</span><span class="fl">0.1</span>) <span class="co"># i.e., deme sizes are all = 1</span></span></code></pre></div>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" tabindex="-1"></a><span class="fu">Fig3_17</span>(<span class="dv">10</span><span class="sc">^</span><span class="dv">1</span>) <span class="co"># i.e., deme sizes are all = 10</span></span></code></pre></div>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" tabindex="-1"></a><span class="fu">Fig3_17</span>(<span class="dv">10</span><span class="sc">^</span><span class="dv">2</span>) <span class="co"># i.e., deme sizes are all = 100</span></span></code></pre></div>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" tabindex="-1"></a><span class="fu">Fig3_17</span>(<span class="dv">10</span><span class="sc">^</span><span class="dv">3</span>) <span class="co"># i.e., deme sizes are all = 1,000</span></span></code></pre></div>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" tabindex="-1"></a><span class="fu">Fig3_17</span>(<span class="dv">10</span><span class="sc">^</span><span class="dv">4</span>) <span class="co"># i.e., deme sizes are all = 10,000</span></span></code></pre></div>
</div>
<div id="deterministic-simulation-of-genetic-drift" class="section level4">
<h4>Deterministic simulation of genetic drift</h4>
<p>The default is that there are five individuals per deme, so <span class="math inline">\(2 \times 5=10\)</span> alleles. You will be asked
how many allele at the dawn of time are A alleles. Simplest is to answer
5.</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" tabindex="-1"></a><span class="fu">Fig3_20</span>() <span class="co"># and answer 5 for number of A alleles</span></span></code></pre></div>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" tabindex="-1"></a><span class="co"># This one will &quot;drift slower&quot;, but you would still be able to</span></span>
<span id="cb21-2"><a href="#cb21-2" tabindex="-1"></a><span class="co"># order the graphs</span></span>
<span id="cb21-3"><a href="#cb21-3" tabindex="-1"></a><span class="fu">Fig3_20</span>(<span class="dv">5</span>) <span class="co"># and answer 10 for number of A alleles</span></span></code></pre></div>
</div>
</div>



<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
